builder/vmware/common/step_run.go

```diff
@@ -87,7 +87,9 @@ func (s *StepRun) Cleanup(state multistep.StateBag) {
 		if running {
 			ui.Say("Stopping virtual machine...")
 			if err := driver.Stop(s.vmxPath); err != nil {
-				ui.Error(fmt.Sprintf("Error stopping VM: %s", err))
+				err = fmt.Errorf("error stopping virtual machine: %v", err)
+				state.Put("error", err)
```

Comment:
As mentioned before, not sure we need to concern ourselves with `error` during cleanup

Reply:
Addressed in https://github.com/hashicorp/packer-plugin-vmware/commit/d6a20f766cb77d80838cf62ada62edaa17649664.